virt/virt-2-4-release-notes.adoc

```diff
@@ -73,6 +73,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 * You can now configure the *Volume Mode* and *Access Mode* for a virtual disk when you xref:../virt/virtual_machines/virt-edit-vms.adoc#virt-vm-add-disk_virt-edit-vms[add a disk to a virtual machine] in the web console. This is also possible when adding a disk to a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[new virtual machine using the wizard].
 
 //CNV-5422 - CNV support for Ceph-based OCS 4.4.
+* Using OpenShift Container Storage with {VirtProductName} allows you to realize the benefits of fault-tolerant storage while retaining the ability to live-migrate between nodes in a cluster.
```

Comment:
This would be a good place to mention the acronym (`OpenShift Container Storage (OCS)`), since it is above the other RN in sequence.

Reply:
Done.